azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -59,10 +63,19 @@
  */
 public class ExecutorManager extends EventHandler implements
     ExecutorManagerAdapter {
+  static final String AZKABAN_QUEUEPROCESSING_ENABLED =
```

Comment:
just out of convention since AZKABAN_QUEUEPROCESSING_ENABLED =
    "azkaban.queueprocessing.enabled" property is exposed via azkaban.properties.